app/models/gobierto_data/dataset.rb

```diff
@@ -32,6 +34,22 @@ def rails_model
                        end
     end
 
+    def create_dataset_from_file(file_path, schema_file: nil, export_files: false, csv_separator: ",")
```

Comment:
I think this name is confusing, because it doesn't create a Dataset instance, but loads a dataset into the database